project/MiMa.scala

```diff
@@ -97,6 +97,50 @@ object MiMa extends AutoPlugin {
     import com.typesafe.tools.mima.core._
 
     val bcIssuesBetween24and25 = Seq(
+
+      // #22387 new materializer
+      FilterAnyProblemStartingWith("akka.stream.impl"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.BidiShape.copyFromPorts"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.BidiShape.reversed"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.MaterializationContext.stageName"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.SinkShape.copyFromPorts"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.Shape.copyFromPorts"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.ClosedShape.copyFromPorts"),
+      ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$FusedGraph$"),
+      ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.Attributes.extractName"),
```

Comment:
I guess this is alright since it was exposing `Module` from `impl`
https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/Attributes.scala#L236